ci/test/02_run_container.sh

```diff
@@ -97,6 +100,7 @@ if [ -z "$DANGER_RUN_CI_ON_HOST" ]; then
                   --mount "${CI_DEPENDS_MOUNT}" \
                   --mount "${CI_DEPENDS_SOURCES_MOUNT}" \
                   --mount "${CI_PREVIOUS_RELEASES_MOUNT}" \
+                  ${CI_BUILD_MOUNT} \
```

Comment:
feel-free-to-disregard: Is there a reason to make `$CI_BUILD_MOUNT` inconsistent (including `--mount`) with the other mount vars?